pastis/matrix_generation/matrix_building_numerical.py

```diff
@@ -734,8 +817,25 @@ def calculate_ref_image(self, save_coro_floor=True, save_psfs=True):
                                                                                                                save_psfs=save_psfs,
                                                                                                                outpath=self.overall_dir)
 
+class MatrixIntensityRST(PastisMatrixIntensities):
+    instrument = 'RST'
+
+    def __int__(self, initial_path='', savepsfs=True, saveopds=True):
+        super().__init__(design=None, savepsfs=savepsfs, saveopds=saveopds)
+
+    def setup_one_pair_function(self):
+        self.calculate_matrix_pair = functools.partial(_rst_matrix_one_pair, self.norm, self.wfe_aber, self.resDir,
+                                                       self.savepsfs, self.saveopds)
+
+    def calculate_ref_image(self, save_coro_floor=False, save_psfs=False, outpath=''):
+        self.contrast_floor, self.norm, self.coro_simulator = calculate_unaberrated_contrast_and_normalization('RST',
+                                                                                                               return_coro_simulator=True,
+                                                                                                               save_coro_floor=save_coro_floor,
+                                                                                                               save_psfs=save_psfs,
+                                                                                                               outpath=outpath)
+
 
 if __name__ == '__main__':
 
         MatrixIntensityLuvoirA(design='small', initial_path=CONFIG_PASTIS.get('local', 'local_data_path')).calc()
-        #MatrixIntensityHicat(initial_path=CONFIG_PASTIS.get('local', 'local_data_path')).calc()
+        #MatrixIntensityHicat(initial_path=CONFIG_PASTIS.get('local', 'local_data_path')).calc()
```

Comment:
Add a blank line at the end of the script.

Reply:
For one time is not my mistake :p